### ISSUE TITLE
Message structure that collapses while editing

### ISSUE THREAD

**Issue by carolinerosin** [haffonist/igamt#2358]
I found it annoying that the message tree collapses every time I edit a segment flavor, is there a way to avoid it ?

When I am editing segment nested deep into groups, after each edit (to set the segment flavor), I need to expand all the tree structure back to my previous state.

**Comment by afrold** [comment]
Yes this is a weakness of the widget we are using. We will do our best to fix this in the future.

**Comment by carolinerosin** [comment]
Otherwise just add an "expand structure" (or something similar) could help. You don't need to expand all, just to see the groups and segments. That way it would only take one click to expand all the groups.
